go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/Inphi/eip4844-interop
 go 1.18
 
 require (
-	github.com/ethereum/go-ethereum v1.10.18
```

Comment:
is this a `go mod tidy` enforced change? I'd think that the version doesn't matter since we replace it with a path dependency [here](https://github.com/Inphi/eip4844-interop/blob/9de255576245f65ee870b535a75a7b71821ff9a2/go.mod#L192)

Reply:
No, I thought you said we should update the go.mod so I made that change manually.  I guess it doesn't hurt though?

Reply:
ah gotcha. Sorry I meant to say the geth submodule was out of date. But this doesn't hurt :-)